src/landscape/components/LandscapeList.test.js

```diff
@@ -101,19 +149,32 @@ const baseListTest = async () => {
   ).toBeInTheDocument();
 
   // Map
-  const mapRegion = screen.getByRole('region', {
-    name: 'Landscapes map',
+  expect(map.addSource).toHaveBeenCalledTimes(3);
+  expect(map.addSource.mock.calls[2][0]).toEqual('landscapes');
+  const geojson = map.addSource.mock.calls[2][1].data;
+  expect(geojson.features.length).toBe(15);
+
+  expect(map.addLayer).toHaveBeenCalledTimes(5);
+  expect(map.addLayer.mock.calls[2][0]).toMatchObject({ id: 'clusters' });
+  expect(map.addLayer.mock.calls[3][0]).toMatchObject({ id: 'cluster-count' });
+  expect(map.addLayer.mock.calls[4][0]).toMatchObject({
+    id: 'unclustered-point',
   });
-  expect(mapRegion).toBeInTheDocument();
 
-  const markers = within(mapRegion).getAllByRole('button');
-  expect(markers.length).toBe(18); // 15 + zoom buttons
-
-  await act(async () => fireEvent.click(markers[0]));
-
-  within(mapRegion).getByRole('link', {
-    name: 'View details about Landscape Name 0',
-  });
+  await act(async () =>
+    events['click:unclustered-point']({
+      features: [geojson.features[0]],
+      lngLat: {
+        lng: 0,
+        lat: 0,
+      },
+    })
+  );
+  const domElement = Popup.setDOMContent.mock.calls[0][0];
+  expect(domElement.querySelector('a').href).toEqual(
+    'http://localhost/landscapes/landscape-0'
```

Comment:
question: should this be localhost or 127.0.0.1?

Reply:
Not, sure, the test is passing so I think that is how the url works for the tests, but I will double check this